db/attachment_test.go

```diff
@@ -453,43 +453,48 @@ func TestForEachStubAttachmentErrors(t *testing.T) {
 	// Call ForEachStubAttachment with invalid attachment; simulates the error scenario.
 	doc := `{"_attachments": "No Attachment"}`
 	assert.NoError(t, base.JSONUnmarshal([]byte(doc), &body))
-	err = db.ForEachStubAttachment(body, 0x1, callback)
+	_, err = body.ToIncomingDocument()
 	assert.Error(t, err, "It should throw 400 Invalid _attachments")
```

Comment:
ToIncomingDocument has changed this error message.

Reply:
It should be the same?

Reply:
It has no underscore